src/App.tsx

```diff
@@ -10,31 +11,31 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState('tab-1');
+
+  const IdSelectedTab = (id: string) => {
+    setSelectedTabId(id);
+  };
```

Comment:
you can name it like `handleTabSelected`
Also don't use naming from uppercase
`idSelectedTab` doesn't sound like name describing that it is function